### PR TITLE
Adding dysgu

### PR DESCRIPTION
**Pull request by kcleal**
Build of dysgu 1.3.16 https://github.com/kcleal/dysgu
dysgu is a SV caller for short or long-read data

### CODE REVIEW

**Comment by kcleal** [comment]
@BiocondaBot please add label

**Review by daler — CHANGES_REQUESTED** [review]

File: recipes/dysgu/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+test:
+  requires:
+    - cython
+  imports:
+    - dysgu
+  files:
+    - dysgu/tests/*
+  commands:
+    - 'pip install cython; dysgu --help'
```
[the remark this answers is not in the document]
Comment (kcleal) [reply recipes/dysgu/meta.yaml:73]: I know its a bit unusual, but cython is required for the runtime. For some read cython is not installed in the test environment. Ive tried several options to try and get it to install such as having a requirements subsection within the test section, but nothing seems to make it install. If I put cython in the run section of requirements, then the lint fails!

File: recipes/dysgu/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+  binary_relocation: False
+  entry_points:
+    - dysgu = dysgu.main:cli
+
+requirements:
+  build:
+    - make
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
```
[the remark this answers is not in the document]
Comment (kcleal) [reply recipes/dysgu/meta.yaml:24]: For some reason the build fails without all of them in the host section. Perhaps something to do with cython needing to know they exists during compilation?

[the remark this answers is not in the document]
Comment (kcleal) [reply recipes/dysgu/meta.yaml:24]: I can try this again, to see if they can be ommited

File: recipes/dysgu/meta.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (kcleal) [reply recipes/dysgu/meta.yaml:73]: Do you know how I can get cython to install in the test section?

[the remark this answers is not in the document]
Comment (daler) [reply recipes/dysgu/meta.yaml:73]: i guess i'm not clear why cython is required at runtime, but if the test fails without it, then the installed package will also be broken. If it's really a runtime dependency for some reason, there is a [lint skip mechanism](https://bioconda.github.io/contributor/linting.html#skipping-persistently-on-a-per-recipe-basis).

[the remark this answers is not in the document]
Comment (kcleal) [reply recipes/dysgu/meta.yaml]: Thanks, this worked!

**Review by daler — APPROVED** [review]

**Comment by daler** [comment]
@BiocondaBot please merge

**Pull request merged**